modules/longhorn/examples/simple/versions.tf

```diff
@@ -11,4 +11,7 @@ terraform {
       version = ">= 2.8.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-kubernetes-addons longhorn example"
+  }
```

Comment:
provider_meta is not required in examples

```suggestion
```

Reply:
addressed